src/Enable.Extensions.Queuing.RabbitMQ/Internal/RabbitMQQueueClient.cs

```diff
@@ -244,8 +254,9 @@ protected override void Dispose(bool disposing)
             {
                 if (disposing)
                 {
+                    _channel.Close();
```

Comment:
Why is this line required? Isn't `Close` called when the `Dispose` is called?

Reply:
https://www.rabbitmq.com/dotnet-api-guide.html#disconnecting Rabbit MQ docs state that close must be called.